server/item/suspicious_stew_type.go

```diff
@@ -0,0 +1,93 @@
+package item
+
+import (
+	"time"
+
+	"github.com/df-mc/dragonfly/server/entity/effect"
+)
+
+type stewEffect uint8
+
+// Uint8 returns the stew as a uint8.
+func (s stewEffect) Uint8() uint8 {
+	return uint8(s)
+}
+
+// StewEffect represents a type of suspicious stew.
+type StewEffect struct {
+	effect effect.Effect
+	stewEffect
+}
+
+// Type returns suspicious stew effect type.
+func (s StewEffect) Type() effect.Effect {
```

Comment:
There's a lot of empty lines in this function and the functions below this one, I'd probably remove them.

Reply:
Also, I'd probably make this a function for `stewEffect` and then switch on the type and then return the respective effects there. I'd also make it return a slice of effects as opposed to a single effect.

Reply:
i thought it could only have an effect at a time

Reply:
Not necessarily, but I imagine it might in the future, so there's no harm in supporting that functionality.

Reply:
I'd also rename `StewEffect` to just `SuspiciousStewType` or `StewType` if you do decide to support multiple effects.

Reply:
gotcha 👍